### PR TITLE
Remove redundant copy from our root readme.

### PR DESCRIPTION
**Pull request by jverkoey**
This content is prone to getting stale and is largely a copy of what's found in docs/docsite-index.md. Instead of copying it into the README.md we now link to the documentation website. We may want to consider increasing the visibility of the documentation link as well.

Pivotal story: https://www.pivotaltracker.com/story/show/156775556

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,12 +62,6 @@ Material Components for iOS uses
 copyright Google Inc. and licensed under
 [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).
 
-Several components use
-[MDFTextAccessibility](https://github.com/material-foundation/material-text-accessibility-ios),
-copyright Google Inc. and licensed under
-[Apache 2.0](https://github.com/material-foundation/material-text-accessibility-ios/blob/master/LICENSE)
-without a NOTICE file.
```
Comment (jverkoey) [inline README.md]: We do not need to attribute dependencies that are pulled in via CocoaPods - CocoaPods will do that for apps.

Comment (romoore) [reply README.md]: Won't Cocoapods pull in the Roboto down below as well? Why is this one special?

Comment (jverkoey) [reply README.md]: I'm not certain what the expectations for Roboto's attributions are as a font, so I've left as-is. Can explore whether we need to attribute it in a follow-up change.

Comment (romoore) [reply README.md]: So the lawyer cats signed-off on removing the MDF stuff but not yet the Roboto stuff?

Comment (jverkoey) [reply README.md]: The lawyer cats never signed on or off of adding the CocoaPods attributions to our README.md. This attribution was added in error.

Comment (romoore) [reply README.md]: +ajsecord@ who added that section explicitly in commit without any other changes.

Comment (jverkoey) [reply README.md]: I've reverted this part of the change so that we're not blocking this PR.

File: README.md [hunk 2]
```diff
@@ -12,48 +12,20 @@ Material Components for iOS are written in Objective-C and support Swift and Int
 
 ## Useful Links
 
+- [Documentation](https://material.io/components/ios/)
```
Comment (romoore) [inline README.md]: Please mark this as `(external site)` to match the links below.

**Review by romoore — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jverkoey) [reply README.md]: Done.

**Pull request merged**